### ISSUE TITLE
Bug: window access in node

### ISSUE THREAD

**Issue by barthy-koeln** [boomworks/payload-plugin-author-fields#19]
This bug prevents me from saving anything. The fields show up fine in the browser, but when the same file is executed in the node environment, the following error occurs:

> [12:44:50] ERROR (payload): ReferenceError: window is not defined
>     at Object.condition (.../payload-plugin-author-fields/dist/addAuthorFields.js:146:38)
>     […]

`window?.location.pathname` gets transpiled to:

```ts
(window === null || window === void 0) ? void 0 : window.location.pathname
```

You cannot even reference `window` this way in node.
You could, however, use `typeof window === 'undefined'` :

```ts
typeof window === 'undefined' ? undefined : window.location.pathname
```

**Comment by ccfiel** [comment]
same issue

```
[00:20:47] ERROR (payload): ReferenceError: window is not defined
    at Object.condition (/Users/chrisianfiel/Projects/lten/node_modules/@boomworks/payload-plugin-author-fields/src/addAuthorFields.ts:160:24)
    at promise (/Users/chrisianfiel/Projects/lten/node_modules/payload/src/fields/hooks/beforeChange/promise.ts:55:19)
    at forEach (/Users/chrisianfiel/Projects/lten/node_modules/payload/src/fields/hooks/beforeChange/traverseFields.ts:46:14)
    at Array.forEach (<anonymous>)
    at traverseFields (/Users/chrisianfiel/Projects/lten/node_modules/payload/src/fields/hooks/beforeChange/traverseFields.ts:44:10)
    at beforeChange (/Users/chrisianfiel/Projects/lten/node_modules/payload/src/fields/hooks/beforeChange/index.ts:37:23)
    at updateByID (/Users/chrisianfiel/Projects/lten/node_modules/payload/src/collections/operations/updateByID.ts:224:36)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async resetLoginAttempts (/Users/chrisianfiel/Projects/lten/node_modules/payload/src/auth/strategies/local/resetLoginAttempts.ts:18:3)
    at async unlock (/Users/chrisianfiel/Projects/lten/node_modules/payload/src/auth/operations/unlock.ts:64:7)
```

**Comment by lindsayevans** [comment]
Thanks for reporting @barthy-koeln - have just published v1.1.5 which should fix this issue
